src/nti/site/_compat.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
```

Comment:
Please don't add this file. It's only used in one place, and that one place doesn't need it.